v-next/hardhat-ethers/src/types.ts

```diff
@@ -0,0 +1,73 @@
+import type { HardhatEthersProvider } from "./internal/hardhat-ethers-provider/hardhat-ethers-provider.js";
```

Comment:
I need a more careful review of the types here, to be sure that they properly align with the V3 artifact implementation

Reply:
I double checked this, I think it looks good.